test/org/sagebionetworks/bridge/services/UserAdminServiceTest.java

```diff
@@ -177,17 +193,22 @@ public void creatingUserThenDeletingRemovesExternalIdAssignment() {
             assertNull(identifier.getHealthCode());
             
             // Now this works
-            externalIdService.assignExternalId(study, externalId, session.getHealthCode());
+            Account account = Account.create();
+            account.setId(BridgeUtils.generateGuid());
+            account.setStudyId(session.getStudyIdentifier().getIdentifier());
+            account.setHealthCode(session.getHealthCode());
```

Comment:
Should we set a different health code, since the original user has been deleted?